src/main/java/com/readup/server/user/domain/UserSocialAccountService.java

```diff
@@ -0,0 +1,28 @@
+package com.readup.server.user.domain;
+
+import org.springframework.stereotype.Service;
+
+import com.readup.server.auth.domain.SocialAccount;
+import com.readup.server.auth.domain.SocialAccountService;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class UserSocialAccountService {
+	private final UserService userService;
+	private final SocialAccountService socialAccountService;
+
+	public User createSocialAccountUser(Long socialAccountId, String nickname) {
+		userService.existsBySocialAccountId(socialAccountId);
+		SocialAccount socialAccount = socialAccountService.getById(socialAccountId);
+
+		User user = User.builder()
```

Comment:
서비스 단에서 User 객체를 빌더로 조합하는 것은 노출 위험이 있습니다 도메인의 정적메서드를 활용하는 것을 추천합니다 :)

Reply:
좋은 조언 감사합니다! 정적메서드 활용하는 방식으로 변경하였습니다.

```suggestion
		User user = User.of(nickname, socialAccount);
```